Xwt/Xwt/MenuItem.cs

```diff
@@ -227,6 +243,12 @@ protected virtual void OnClicked (EventArgs e)
 				base.BackendHost.OnAfterEventRemove (MenuItemEvent.Clicked, clicked);
 			}
 		}
+
+		protected override void Dispose (bool release_all)
+		{
+			Backend.Dispose ();
```

Comment:
We should only Dispose when `release_all` is true, otherwise we're going to be doing the following:

```
MenuItem.Finalizer() -> MacMenuItem.Dispose() -> MacMenuItem.Dispose(true)
```

Which will end up throwing an AppKit inconsistency exception for not being on the UI Thread.